memcached/init.lua

```diff
@@ -89,15 +89,15 @@ void   memcached_handler   (struct memcached_service *p, int fd);
 int    memcached_setsockopt(int fd, const char *family, const char *type);
 ]]
 
-function startswith(str, start)
+local function startswith(str, start)
        return string.sub(str, 1, string.len(start)) == start
 end
 
 local typetable = {
     name = {
         'string',
         function () return 'memcached' end,
-        function (x) return true end,
+        function () return true end,
```

Comment:
(If you want my opinion, I prefer to keep some function contract information — how many arguments it accepts — and use underscore placeholders as opposite to strip unused arguments at all.)

Reply:
> If you want my opinion

I remember your opinion. However, I still don't understand why do you need. Number of arguments accepted by function is useless information without knowledge what kind of arguments function accept. It means that you will go to API doc in any case.

(replaced to underscore for you)

Reply:
Here we have a bunch of function pairs: one accepts zero arguments, another accepts one argument. With this extra information it easier to spot the pattern while looking into those declarations.